src/components/ReactFlagsSelect/ReactFlagsSelect.test.tsx

```diff
@@ -70,6 +70,18 @@ describe("ReactFlagsSelect", () => {
         expect(screen.getByText("GB")).toBeInTheDocument();
       });
 
+      it("renders a special custom label of the valid selected country if present", () => {
```

Comment:
Can we rephrase this to emphasize that we are testing for a custom label with value as object.
for example - `renders a custom label of the valid selected country if present as a CustomLabel object`.
Hence the previous test should be `... if present as a string`